packages/hoppscotch-common/src/pages/import.vue

```diff
@@ -79,6 +79,7 @@ const importCollections = (url: unknown, type: unknown) =>
         content.data,
         TO.fromPredicate(isOfType("string")),
         TE.fromTaskOption(() => IMPORTER_INVALID_FILE_FORMAT),
+        // AM-COMMENT: ask james if this got introduced in any of the multiple import PRs
```

Comment:
Is there anything to handle here? This file wasn't touched as far as I remember while implementing multiple file imports.

Reply:
this `import.vue` file handles the imports from URLs. since the multiple import PR, all our importers now accept arrays. i think that change was not added here. will need to test and make a hotfix.

Reply:
Gotcha; don't remember coming across any inconsistencies while testing the import from URL flows (Gist & Open API). Import from Gist now supports importing multiple files; will keep an eye out.